requirements.txt

```diff
@@ -8,4 +8,5 @@ ledgercomm[hid]
 rich==13.3.4
 argcomplete==3.2.2
 
-multiversx-sdk[ledger]==1.1.0
+# multiversx-sdk[ledger]==1.1.0
```

Comment:
We should not forget about this (should be fixed right before release).

Reply:
indeed, it will